src/uu/expand/src/expand.rs

```diff
@@ -89,23 +89,36 @@ fn tabstops_parse(s: &str) -> (RemainingMode, Vec<usize>) {
                 }
                 _ => {
                     // Parse a number from the byte sequence.
-                    let num = from_utf8(&bytes[i..]).unwrap().parse::<usize>().unwrap();
+                    let s = from_utf8(&bytes[i..]).unwrap();
+                    if let Ok(num) = s.parse::<usize>() {
+                        // Tab size must be positive.
+                        if num == 0 {
+                            crash!(1, "tab size cannot be 0");
+                        }
 
-                    // Tab size must be positive.
-                    if num == 0 {
-                        crash!(1, "tab size cannot be 0");
-                    }
+                        // Tab sizes must be ascending.
+                        if let Some(last_stop) = nums.last() {
+                            if *last_stop >= num {
+                                crash!(1, "tab sizes must be ascending");
+                            }
+                        }
 
-                    // Tab sizes must be ascending.
-                    if let Some(last_stop) = nums.last() {
-                        if *last_stop >= num {
-                            crash!(1, "tab sizes must be ascending");
+                        // Append this tab stop to the list of all tabstops.
+                        nums.push(num);
+                        break;
+                    } else {
+                        let s = s.trim_start_matches(char::is_numeric);
+                        if s.starts_with('/') || s.starts_with('+') {
+                            crash!(
+                                1,
+                                "{} specifier not at start of number: {}",
```

Comment:
is the same error message as gnu?
seems that the test isn't going through that line

Reply:
Yes, it is the same message as used by GNU expand.

> seems that the test isn't going through that line

What do you mean with this?